tests/client_integration_test.go

```diff
@@ -73,11 +78,12 @@ func newTestController(t *testing.T) testController {
 
 	ctx, cancel := context.WithCancel(context.Background())
 	cmd := exec.CommandContext(ctx, serverBinaryPath,
-		"-d",
-		tempdir,
-		"-i",
-		address,
-		"-c", cert, "-k", key)
+		"-d", tempdir,
```

Comment:
The ordering of these flags forms an...unintentional word choice I didn't notice until now. Should probably switch it 😅

Reply:
LOL done.